config/default.yml

```diff
@@ -861,6 +861,13 @@ Style/MultilineOperationIndentation:
   # But it can be overridden by setting this parameter
   IndentationWidth: ~
 
+Style/NegatedIf:
+  EnforcedStyle: both
+  SupportedStyles:
```

Comment:
:+1: Is this enough? There seems to be some inconsistency in how much of a description to give—some almost duplicate the documentation in the cop itself…